hi-ml-multimodal/src/health_multimodal/image/model/resnet.py

```diff
@@ -20,7 +22,9 @@ class ResNetHIML(ResNet):
     def __init__(self, **kwargs: Any) -> None:
         super().__init__(**kwargs)
 
-    def forward(self, x: torch.Tensor) -> torch.Tensor:
+    def forward(self, x: torch.Tensor,
+                return_skip: bool = False) -> Union[torch.Tensor, TypeSkipConnections]:
```

Comment:
I think it would be helpful to add a docstring here that describes the arguments and in particular the returned tuple in detail.